src/code/audio_seqplayer.c

```diff
@@ -11,6 +11,25 @@
 
 extern u8 D_80130470[];
 
+// static u8 D_80130470[] = {
+// /* 0x00 */ 0x89, 0xC0, 0x85, 0x84, 0x82, 0x77, 0x80, 0x9F, 0x80, 0x01, 0x80, 0x9F, 0x82, 0x77, 0x85, 0x84,
+// /* 0x10 */ 0x89, 0xC0, 0x8F, 0x1F, 0x95, 0x94, 0x9D, 0x0F, 0xA5, 0x7F, 0xAE, 0xCD, 0xB8, 0xE4, 0xC3, 0xAA,
+// /* 0x20 */ 0xCF, 0x05, 0xDA, 0xD9, 0xE7, 0x08, 0xF3, 0x75, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
```

Comment:
if this is already migrated to C, id prob just use it and keep it non static. assuming thats the only reason its commented out rn

Reply:
looking at it further im pretty confused as to whats going on, is this data part of another file thats not split yet?